src/components/profile/ReportingStructure.vue

```diff
@@ -95,10 +96,24 @@ export default {
   created() {
     this.update_user();
   },
+  mounted() {
+    window.setTimeout(() => {
+      const { grid, gridContainer } = this.$refs;
+
+      if (
+        grid &&
+        gridContainer &&
+        grid.clientHeight > gridContainer.clientHeight
+      ) {
+        this.managesExpandable = true;
+      }
+    }, 700);
+  },
   data() {
     return {
       related: new Related(),
-      initiallyShown: 10,
+      managesExpanded: false,
```

Comment:
these break between profiles. Both are not reset when switching to another profile.